src/proc_image_processing/cpu/filters/detectors/fence_detector.h

```diff
@@ -200,38 +200,38 @@ namespace proc_image_processing {
           }
 
           if (bar_founded == 2) {
-            int x = 0, y = 0;
-            // X from bottom bar plus
-            x = (rect_from_hori_bar.center.x +
-              (final_vert_bar[0].x + final_vert_bar[1].x) / 2) /
-              2;
-
-            y = (y_coord_from_bottom + final_vert_bar[0].y +
-              final_vert_bar[1].y) /
-              3;
-            center = cv::Point(x, y);
-            if (debug_contour_()) {
-              cv::circle(output_image_, center, 5, CV_RGB(0, 255, 255), 20);
-            }
-            SetCameraOffset(center, image.rows, image.cols);
-            fence.SetCenter(center);
+              int x = 0, y = 0;
```

Comment:
Are these the formatting generated by CLion you were talking about during the scrum?

Reply:
No refactor has been made in this branch. Refactor is in the apply-style-guide branch.